website/docs/reference/warehouse-setups/trino-setup.md

```diff
@@ -65,69 +137,181 @@ trino:
-| session_properties             | Sets Trino session properties used in the connection                                                         | Optional                                                                                                | `query_max_run_time: 4h`         |
-| database                       | Specify the database to build models into                                                                    | Required                                                                                                | `analytics`                      |
-| schema                         | Specify the schema to build models into. Note: it is not recommended to use upper or mixed case schema names | Required                                                                                                | `public`                         |
-| host                           | The hostname to connect to                                                                                   | Required                                                                                                | `127.0.0.1`                      |
-| port                           | The port to connect to the host on                                                                           | Required                                                                                                | `8080`                           |
-| threads                        | How many threads dbt should use                                                                              | Optional (default is `1`)                                                                               | `8`                              |
-| prepared_statements_enabled    | Enable usage of Trino prepared statements (used in `dbt seed` commands)                                      | Optional (default is `true`)                                                                            | `true` or `false`                |
-| retries                        | Configure how many times a database operation is retried when connection issues arise                        | Optional (default is `3`)                                                                               | `10`                             |
-| timezone                       | The time zone for the Trino session                                                                          | Optional (defaults to the client side local timezone)                                                   | `Europe/Brussels`                |
+</TabItem>
+
+<TabItem value="oauth">
+
+Authenticating in dbt-core using OAuth 2 is currently underdocumented, but helpful information is given on the [trino-python-client's README](https://github.com/trinodb/trino-python-client#oauth2-authentication).
+
+For addiontal information, refer to Trino's doc page on [Oauth2 Authentication](https://trino.io/docs/current/security/oauth2.html)
```

Comment:
@monimiller @hovaesco do either of y'all:
1. know the required parameters for Trino auth, or
2. have a sample `profiles.yml` for it?

Looking directly at the `TrinoOauthCredentials` class ([source](https://github.com/starburstdata/dbt-trino/blob/0ca08ab0a32f615f2180cd8e40ef117ec02fe941/dbt/adapters/trino/connections.py#L248-L273)), I don't understand what the default URL even is...

Reply:
btw I just tried OAuth and couldn't get it to work. opened this https://github.com/starburstdata/dbt-trino/issues/275